src/Loop.php

```diff
@@ -47,4 +47,154 @@ public static function set(LoopInterface $loop)
     {
         self::$instance = $loop;
     }
+
+    /**
+     * [Advanced] Register a listener to be notified when a stream is ready to read.
```

Comment:
Why are only adding streams marked as advanced? IMHO if we mark those two we should also mark `removeReadStream` && `removeWriteStream` as such. And maybe mention [react/stream](https://reactphp.org/stream/) for easier usage.

Reply:
Fair point. I've applied the exact same documentation from the `LoopInterface::addReadStream()` method:

https://github.com/reactphp/event-loop/blob/1a709e2847cbd6f61110349792239fde283f4edc/src/LoopInterface.php#L8

Your suggestion sounds perfectly reasonable, perhaps file this as a follow-up PR to also adjust the interface and the README (similar to #110)? :+1:

Reply:
Will do 👍